app.js

```diff
@@ -125,9 +126,16 @@ app.get('/alerts', async function (req, res)
 
                 object.entity[entityIndex].alert.informedEntity.forEach(idPair =>
                 {
+
+                    const route = routes.find(route => {
+                        return route.route_id === idPair.routeId;
+                      });
+                      
+                    const routeShortName = route?.route_short_name ?? "Unknown";

```

Comment:
```suggestion
                    const routeShortName = route?.route_short_name ?? 'Unknown';
```

Single quotes for strings, please.